src/command.ts

```diff
@@ -371,7 +380,7 @@ export class Command {
    * @return an async function that executes the command.
    */
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  runner(): (...a: any[]) => Promise<void> {
+  runner(): (...a: any[]) => Promise<any> {
```

Comment:
I'm assuming this change is unrelated to the changes above?

Reply:
This is actually the 'result' that we're printing out in the main part of this change. 

This was incorrectly typed before - runner() is actually returning whatever is returned by the command's action function.

Reply:
Got it. Thanks!